pom.xml

```diff
@@ -44,10 +44,52 @@
             <artifactId>log4j-api</artifactId>
             <version>2.19.0</version>
         </dependency>
+
+        <dependency>
+            <groupId>org.openjdk.jol</groupId>
+            <artifactId>jol-core</artifactId>
+            <version>0.16</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.openjdk.jmh</groupId>
+            <artifactId>jmh-core</artifactId>
+            <version>1.36</version>
```

Comment:
Ideally this version would be a property to make upgrading the dependency a single line of change; otherwise, there's a risk that someone will update this dependency but leave the one below to an older version.